docs/root/development/performance/cpu_battery_impact.rst

```diff
@@ -0,0 +1,150 @@
+.. _dev_performance_cpu_battery:
+
+Analysis of CPU/battery impact
+==============================
+
+Modified versions of the "hello world" example apps were used to run these experiments:
+
+- `Android control app <https://github.com/lyft/envoy-mobile/tree/ac/envoy-battery-cpu-branch/examples/kotlin/control>`_
+- `Android Envoy app <https://github.com/lyft/envoy-mobile/tree/ac/envoy-battery-cpu-branch/examples/kotlin/hello_world>`_
+- `iOS control app <https://github.com/lyft/envoy-mobile/tree/ac/envoy-battery-cpu-branch/examples/objective-c/control/control>`_
+- `iOS Envoy app <https://github.com/lyft/envoy-mobile/tree/ac/envoy-battery-cpu-branch/examples/objective-c/xcode_variant/EnvoyObjc/EnvoyObjc>`_
+
+- **Control** - Made a request every ``200ms`` to an endpoint without Envoy compiled in the app
+- **Envoy** - Made the same request at the same interval, but routed through an instance of Envoy
+
+All request/response caching was disabled.
+
+Results
+~~~~~~~
+
+iOS
+---
+
```

Comment:
Would be good to say up to what commit these results are representative. So that we know what these results are tied to. I mentioned that in the [binary size](https://envoy-mobile.github.io/docs/envoy-mobile/latest/development/performance/binary_size.html#current-status) docs.

Reply:
Good call